### PR TITLE
DOM: Add mobile menu

### PR DESCRIPTION
**Pull request by Nedramrez**
- Change linter files
- Add function that makes the mobile menu appear
- Add scrtpt for the mobile menu
- Add function that makes the mobile menu disappear (partner's commit)
- Add styling to mobile menu

### CODE REVIEW

File: assets/css/styles.css [hunk 1]
```diff
@@ -22,6 +22,45 @@ body {
...
+  font-size: 35px;
+  line-height: 44px;
+  color: white;
+  display: flex;
+  flex-direction: column;
+  text-decoration: none;
+  padding: 0 0 18px 25px;
+}
+
+.mobile-menu-close {
```
Comment (Munsa1) [inline assets/css/styles.css:54]: - [x] There is a issue with the close icon in mobie size. At `375 px` the button behaves correctly and is aligned properly. At `320 px` the close button chages size and dpes not work. At `425 px` the close button dissapears altogether. `425 px`is for large mobile phones and `320 px` is for small phones. Your design should be compatible for all these versions, kindly fix it

#### 320 px
image

####  375 px
image

#### 425 px
image

File: assets/javascript/script.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+  const appearMobileUlLi3 = document.createElement('li');
+  appearMobileUlLi3.classList.add('mobile-list-item');
+  appearMobileUl.appendChild(appearMobileUlLi3);
+
+  const appearMobileLink3 = document.createElement('a');
+  appearMobileLink3.classList.add('mobile-link');
+  appearMobileUlLi3.appendChild(appearMobileLink3);
+  appearMobileLink3.setAttribute('href', '#Contact');
+  appearMobileLink3.textContent = 'Contact';
```
Comment (Munsa1) [inline assets/javascript/script.js]: - [x] Each of the links should take the user to the specific part of the page when clicked. 

image

- The Portfolio link in the image should take the user to the `Portfolio` part of the web page
- The About link in the image should also take the user to the `About` part of the web page.
- The Contact link in the mobile menu should also take the user to the `Contact`, part of the page.

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @Nedramrez  @DaveZag , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
:heavy_check_mark: Descriptive Readme
:heavy_check_mark: Passing Linters
:heavy_check_mark: Correct Gitflow
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by zhadier — APPROVED** [review]
## Hi @Nedramrez  👋  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 



# Congratulations! 🎉 

✔️ No linter errors
✔️ Professional pull request title and description

### Keep Rocking!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**